src/annotator/integrations/html-metadata.js

```diff
@@ -287,22 +259,24 @@ export class HTMLMetadata {
         encodeURIComponent(dcUrnRelationComponent) +
         '/' +
         encodeURIComponent(dcUrnIdentifierComponent);
-      this.metadata.link.push({ href: dcUrn });
-      // set this as the documentFingerprint as a hint to include this in search queries
-      this.metadata.documentFingerprint = dcUrn;
```

Comment:
The `documentFingerprint` assignment was moved into the `getDocumentMetadata` method.